crates/humanode-runtime/src/frontier_precompiles.rs

```diff
@@ -120,8 +122,11 @@ where
         }
     }
 
-    fn is_precompile(&self, address: H160) -> bool {
-        Self::used_addresses().contains(&address)
+    fn is_precompile(&self, address: H160, _gas: u64) -> IsPrecompileResult {
+        IsPrecompileResult::Answer {
```

Comment:
Let's create an issue to do this properly

Reply:
https://github.com/humanode-network/humanode/issues

Reply:
Mind clarifying the issue number here?

Reply:
https://github.com/humanode-network/humanode/issues/863